.github/workflows/release-please-check.yaml

```diff
@@ -23,6 +23,5 @@ jobs:
       - uses: googleapis/release-please-action@v4
         id: release-please
         with:
-          token: ${{ secrets.AGNTCY_BUILD_BOT_GH_TOKEN }}
```

Comment:
Please let the this token here. The buildbot user should cut a tag upon changes.